Project-Service-implementation-Script/main.py

```diff
@@ -0,0 +1,5238 @@
+                                      "BODY : " + str(surveySolutionCreationReqBody),
+                                      "Status code : " + str(responseCreateSolutionApi.status_code),
+                                      "Response : " + responseCreateSolutionApi.text]
+                        fileheader = [surveySolutionCreationReqBody['name'].encode('utf-8').decode('utf-8'),'Program Sheet Validation'," "]
+                        createAPILog(parentFolder, messageArr)
+                        apicheckslog(parentFolder,fileheader)
+                        if responseCreateSolutionApi.status_code == 200:
+                            responseCreateSolutionApi = responseCreateSolutionApi.json()
+                            urlSearchSolution = config.get(environment, 'INTERNAL_KONG_IP') + config.get(environment,'fetchSolutionDetails') + "survey&page=1&limit=10&search=" + str(surveySolutionExternalId)
+                            responseSearchSolution = requests.request("POST", urlSearchSolution,
+                                                                      headers=headerCreateSolutionApi)
+                            messageArr = ["********* Search Survey Solution *********", "URL : " + urlSearchSolution,
+                                          "Status code : " + str(responseSearchSolution.status_code),
+                                          "Response : " + responseSearchSolution.text]
+                            createAPILog(parentFolder, messageArr)
+                            # apicheckslog(parentFolder, messageArr)
+                            if responseSearchSolution.status_code == 200:
+                                responseSearchSolutionApi = responseSearchSolution.json()
+                                surveySolutionExternalId = None
+                                surveySolutionExternalId = responseSearchSolutionApi['result']['data'][0]['externalId']
+                            else:
+                                print("Solution fetch API failed")
+                                print("URL : " + urlSearchSolution)
+                                terminatingMessage("Status Code : " + responseSearchSolution.status_code)
+
+                            solutionId = None
+                            solutionId = responseCreateSolutionApi["result"]["solutionId"]
+                            bodySolutionUpdate = {"creator": dictDetailsEnv['Name_of_the_creator'].encode('utf-8').decode('utf-8')}
+                            solutionUpdate(parentFolder, accessToken, solutionId, bodySolutionUpdate)
+                            return [solutionId, surveySolutionExternalId]
+                        else:
+                            terminatingMessage("Survey creation Failed, check logs!")
+
+# upload survey questions 
+def uploadSurveyQuestions(parentFolder, wbSurvey, addObservationSolution, accessToken, surveySolutionExternalId, surveyParentSolutionId,millisecond):
```

Comment:
@MallanagoudaB We clearly discussed that we have to remove observation and survey related code if it is not required. this script only for project-service. why this is not taken care?